views/pages/index.ejs

```diff
@@ -47,30 +47,57 @@
         <div id="about-us-image" class="col-md-6"></div>
       </div>
     </section>
-    <section id="sponsors">
+    <section id="sponsors-events">
```

Comment:
Could we have the events on our website linked back to the event on the Facebook page?

Reply:
They already are, will look into making it a bit more obvious